### DISCUSSION TITLE
刷宇宙相关玩法总结

### DISCUSSION THREAD

**Discussion by foodtooth** [LmeSzinc/StarRailCopilot#34] [Ideas]
欢迎讨论

1. 基本思路仍然是，手动配队，配置队伍编号和要打的模拟宇宙编号
2. 宇宙的机制目前是前100个精英给掉落物，非常丰厚，但是1精英给一份，2精英给两份，关底给3份
    1. 最佳策略自然是打各个宇宙的最高难度，还能auto稳定通关，但是感觉有些难度
    2. 次选就是，或最高难度选择尽量打精英，失败只能重复再打1精英；或降低难度全通。具体掉落差异没看过
3. 脚本跑不用考虑流派的花样，以稳定为主，应该是宇宙4丰饶，其它都是存护
4. 祝福选不到对应流派时候选项也有一定的相性，但是我觉得初版可以不用考虑，留个接口就好
---
5. 每个人会有自己的队伍以及觉得可以应对的宇宙和难度，我们只维护下面说的相关性表格。如果整体来讲，宇宙6的敌人机制最简单，拿存护或者巡猎打应该都很稳

**Comment by BeatSeat** [comment]
宇宙的机制好像是一共给200份掉落物这样的，理论上也可以直接刷1精英，但是考虑到祝福其实在后面优势越来越大其实尽量往后打打最好。然后我觉得直接自己选命途就可以了，毕竟也有可能队伍练度比较高单纯想速刷，只需要打每关最后一个怪或者优先走事件（其实比如宇宙三我经常存护过不去走记忆就能过）。

当然这块还有一些可能未来需要的选择提供，比如走不走遭遇、优先事件还是战斗什么的，毕竟也有可能刷宇宙是用来刷事件刷奇物的。

关于祝福和流派之类，其实我觉得可以参考一下[隔壁项目](https://github.com/CHNZYX/Auto_Simulated_Universe)的相关经验，比如有一个优先级，优先选择一部分祝福（例如火堆外的夜、戒律性闪变），其次选择当前命途，如果选不到就一直刷新，然后考虑一个命途的优先级，比如存护毁灭>丰饶>巡猎>欢愉记忆之类的。

**Reply by foodtooth** [reply]
不是200份，已经不少人测了发出来是会有100次的掉落，然后两个精英和首领的掉落份数不同。所以才有那个收益的问题。

分支选择、奇物选择、非当前命途的祝福选择，这些可变性有点大。。可以大家维护个相关度表格出来

**Comment by BeatSeat** [comment]
从隔壁复制过来的各个命途的优先祝福列表：
存护：['零维强化','均晶转变','共晶反应','宏观偏析','超静定场','谐振传递','四棱锥体','聚塑','哨戒','亚共晶体','切变结构','弥合','迸裂晶格']

记忆：['体验的富翁','全面记忆','第二次初恋','浮黎','缄默','纯真','难言的羞耻','怅然若失','麻木不仁','不寒而栗','特立独行','头晕目眩','多愁善感','沦浃肌髓']

虚无：['苦难与阳光','怀疑的四重根','局外人','为何一切尚未消失','感官追奉者的葬礼','被装在套子里的人','旷野的呼告','存在的黄昏','火堆外的夜','知觉迷墙','虚妄贡品','日出之前','无根据颂歌','自欺咖啡馆','他人即地狱','开端与终结']

丰饶：['诸行无常','诸法无我','一法界心','施诸愿印','延彼遐龄','厌离邪秽苦','天人不动众','宝光烛日月','明澈琉璃身','法雨','胜军','灭罪累生善']

巡猎：['柘弓危矢','射不主皮','帝星君临','白矢决射御','云镝逐步离','彤弓素矰','背孤击虚']

毁灭：['极端氦闪','激变变星','事件视界','寰宇热寂特征数','反物质非逆方程','戒律性闪变','危害性余光','毁灭性吸积','原生黑洞','轨道红移','预兆性景深','递增性末日','灾难性共振','破坏性耀发','偏振受体','永坍缩体','不稳定带','哨戒卫星','回光效应']

欢愉：['末日狂欢','开盖有奖','茫茫白夜','众生安眠','阴风阵阵','被涂污的信天翁','十二猴子与怒汉','操行满分','基本有害','灰暗的火','第二十一条军规','流吧你的眼泪']

其中每个列表的前三个是命途回响，可供大家做一个参考（这里直接复制过来的，所以也不是完整名字）选择机制是优先从前往后选择对应命途的上表所示祝福，其次是对应命途的祝福，否则就刷新祝福。

另外还有一些通用优先的：火堆外的夜。

具体的表格应该列在群里吗？

**Reply by foodtooth** [reply]
可以在线文档，方便一起编辑。

另外，感觉lme想一步到胃。。 😃 也可以根据已有选择（命途、奇物、祝福等所有可选要素），判断新的祝福和奇物的相关度
